shell/nuxt/App.js

```diff
@@ -0,0 +1,210 @@
+import Vue from 'vue'
+import { decode, parsePath, withoutBase, withoutTrailingSlash, normalizeURL } from 'ufo'
+
+import { getMatchedComponentsInstances, getChildrenComponentInstancesUsingFetch, promisify, globalHandleError, urlJoin, sanitizeComponent } from './utils'
+import NuxtError from '../layouts/error.vue'
+import NuxtLoading from '../components/nav/GlobalLoading.vue'
+import NuxtBuildIndicator from './components/nuxt-build-indicator'
+
+import '../assets/styles/app.scss'
+
+import _77180f1e from '../layouts/blank.vue'
```

Comment:
I have some concerns about hardcoding hashed imports, as they may cause cache issues.
Let's make an issue to review this and apply proper naming maybe?